cockatrice/src/sequenceEdit/shortcutstab.cpp

```diff
@@ -16,3 +23,33 @@ ShortcutsTab::~ShortcutsTab()
 {
     delete ui;
 }
+
+void ShortcutsTab::resetShortcuts()
+{
+    settingsCache->shortcuts().resetAllShortcuts();
+}
+
+void ShortcutsTab::refreshEdits()
+{
+    QList<SequenceEdit*> edits = this->findChildren<SequenceEdit*>();
+    for(int i= 0; i < edits.length(); ++i)
+    {
+        edits.at(i)->refreshShortcut();
+    }
+    QMessageBox::information(this,tr("Shortcuts reset"),tr("All shortcuts have been reset"));
+}
+
+void ShortcutsTab::clearShortcuts()
+{
+    settingsCache->shortcuts().clearAllShortcuts();
+}
+
+void ShortcutsTab::afterClear()
+{
+    QList<SequenceEdit*> edits = this->findChildren<SequenceEdit*>();
+    for(int i= 0; i < edits.length(); ++i)
+    {
+        edits.at(i)->clear();
+    }
+    QMessageBox::information(this,tr("Shortcuts reset"),tr("All shortcuts have been cleared"));
```

Comment:
We have 2 messages boxes here with the same name/title description, but they do two completely different things.

Other languages use several terms for this different actions.
That will bring some trouble for translators and makes the UI less simple.